trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -28,11 +28,28 @@ const getCompany = async (req, res) => {
   }
 };
 
-const getPartyUrn = async ({ companyRegNo }) => {
+const getPartyUrn = async ({ companyRegNo, companyName }) => {
   if (!companyRegNo) {
     return '';
   }
 
+  if (process.env.AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED === 'true') {
+    let partyDbInfo = await api.getPartyDbInfo({ companyRegNo });
+
+    if (partyDbInfo.status === 404) {
+      if (!companyName) {
+        return '';
+      }
+
+      partyDbInfo = await api.createParty({ companyRegNo, companyName });
+    }
+
+    if (partyDbInfo.status !== 200) {
+      return '';
+    }
+
+    return partyDbInfo.data[0].partyUrn || '';
```

Comment:
This is accessing data from getPartyDbInfo differently to line 58 which is calling the same thing, is this going to cause any issues?

Reply:
Ah I saw the other part of this and left a comment there instead